lib/vdsm/virt/thinp.py

```diff
@@ -462,7 +462,16 @@ def callback(error=None):
         # Note that the volume is extended after the replica is extended, so
```

Comment:
Are comments in lines 462-463 still relevant?

Reply:
Yes, total time still includes both extend volume and extend replica times.